File: Trabalho1/Board.java
public class Board {
    private int[][] table;
    private int blank_row;
    private int blank_col;

    Board(int[][] b, int br, int bc){
        table = b;
        blank_row = br;
        blank_col = bc;
    }

    int getBlankRow(){
        return blank_row;
    }

    int getBlankCol(){
        return blank_col;
    }

    int[][] getTable(){
        return table;
    }

    // void setBlankRow(int br){
    //     blank_row = br;
    // }

    // void setBlankCol(int bc){
    //     blank_col = bc;
    // }
}

File: Trabalho1/board.py
class Board:
    def __init__(self, table, blank_row, blank_col):
        self.table = table
        self.blank_row = blank_row
        self.blank_col = blank_col
        self.n = 0

    def getTable(self):
        return self.table
    
    def getBlankRow(self):
        return self.blank_row
    
    def getBlankCol(self):
        return self.blank_col
    
    def __lt__(self, other):
        return self.n <= other.n